.ci/releasePipeline.groovy

```diff
@@ -249,17 +249,30 @@ def pushToDtRepository(Map args = [source: null, dest: null]) {
                                           [envVar: 'username', vaultKey: 'username', isRequired: true],
                                           [envVar: 'password', vaultKey: 'password', isRequired: true]]]]
             ) {
-                sh '''
+                sh(label: "push to DT artifactory", script: '''
                     curl --request PUT $repo_path/$dest
                          --user "$username":"$password"
                          --fail-with-body
                          --upload-file $source
-                   '''.replaceAll("\n", " ").replaceAll(/ +/, " ")
+                   '''.replaceAll("\n", " ").replaceAll(/ +/, " "))
             }
         }
     }
 }
 
+void sbom(Context ctx) {
+    stage("Generate SBOM") {
+        def sbomTools = load(".ci/jenkins/tools/sbom.groovy")
+
+        sbomTools.install()
```

Comment:
nit: since the string "sbom.cdx.json" is used 5 times, maybe it's worth creating a constant for that?